backend/api/Database/Models/AssetDeck.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+using Microsoft.EntityFrameworkCore;
+
+#nullable disable
```

Comment:
Is this needed when all properties are required?

Reply:
The constructor functions still need it